components/button.html

```diff
@@ -0,0 +1,156 @@
+
+
+                            <li class="btn_inner pad"> <button class="tertiary_btn ">
+                                    <div class="btn_flex">
+                                        <span class="material-icons">
+                                            notifications
+                                        </span>
+                                        <div>Tertiary Link</div>
+                                    </div>
+                                </button> </li>
+                        </ul>
+                    </div>
+                    <div class="carbon-img1"><iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2831%2C129%2C109%2C1%29&t=night-owl&wt=none&l=auto&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=false&pv=74px&ph=72px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522primary_btn%2520%2522%253E%253Cdiv%2520class%253D%2522btn_flex%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2520%2522%253Efavorite%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%253EPrimary%2520Link%253C%252Fdiv%253E%253C%252Fdiv%253E%253C%252Fbutton%253E%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522secondary_btn%2520%2522%253E%253Cdiv%2520class%253D%2522btn_flex%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522materialicons%2522%253Eshopping_cart%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%253ESecondary%2520Link%253C%252Fdiv%253E%253C%252Fdiv%253E%253C%252Fbutton%253E%2520%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522tertiary_btn%2520%2522%253E%2520%253Cdiv%2520class%253D%2522btn_flex%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2522%253E%2520notifications%2520%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%253ETertiary%2520Link%253C%252Fdiv%253E%253C%252Fdiv%253E%2520%253C%252Fbutton%253E%2520%253C%252Fli%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+                        style="width: 773px; height: 452px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                      </iframe></div>
+
+
+
+                    <h1 class="mer">Floating Action Button</h1>
```

Comment:
Hey!
You could also add `cursor: pointer` as it considered to be better in terms of UI when the user is scrolling through.

Reply:
yes! I guess I missed it.